modules/core/src/main/java/com/google/refine/importers/ImportingParserBase.java

```diff
@@ -103,6 +104,12 @@ public void parse(Project project, ProjectMetadata metadata,
                 break;
             }
         }
+
+        // if user doesn't choose 'storeBlankColumns', delete all empty columns.
```

Comment:
```suggestion
        // if user set 'storeBlankColumns' to false, delete all empty columns.
```
Just a small rewording to make it clearer what happens if they take the default and don't set anything